cime_config/buildnml

```diff
@@ -189,6 +190,7 @@ def buildnml(case, caseroot, compname):
         config["blom_coupling"] = blom_coupling
         config["blom_tracer_modules"] = blom_tracer_modules
         config["blom_atrc"] = blom_atrc
+        config["hamocc_output_size"] = hamocc_output_size if is_test == False else 'dummy' # choose is_test, if is_test==True  
```

Comment:
I'm wondering if this is necessary - the parser will have a logic of taking the last match that is found? At least otherwise, the existing `is_test` and `empty_hist` logic wouldn't work either.